src/components/tx-flow/flows/SignMessage/SignMessage.tsx

```diff
@@ -256,6 +276,8 @@ const SignMessage = ({ message, safeAppId, requestId }: ProposeProps | ConfirmPr
             <WrongChainWarning />
 
             <MessageDialogError isOwner={isOwner} submitError={submitError} />
+
+            <RiskConfirmationError />
```

Comment:
Similar to the above comment, this currently says `I understand the risks and would like to continue this transaction` but `message` or something similar would make more sense imo.